elements/pfe-button/demo/pfe-button.html

```diff
@@ -5,175 +5,90 @@
   </pfe-button>
 </section>
 
 <section>
   <h2>Disabled</h2>
-  <pfe-button disabled><button>Primary disabled</button></pfe-button>
-  <pfe-button disabled variant="secondary"><button>Secondary disabled</button></pfe-button>
-  <pfe-button disabled variant="secondary" danger><button>Danger secondary disabled</button></pfe-button>
-  <pfe-button disabled variant="tertiary"><button>Tertiary disabled</button></pfe-button>
-  <pfe-button disabled danger><button>Danger disabled</button></pfe-button>
-  <pfe-button disabled warning><button>Warning disabled</button></pfe-button>
+  <pfe-button disabled>Primary disabled</pfe-button>
+  <pfe-button disabled variant="secondary">Secondary disabled</pfe-button>
+  <pfe-button disabled variant="secondary" danger>Danger secondary disabled</pfe-button>
+  <pfe-button disabled variant="tertiary">Tertiary disabled</pfe-button>
+  <pfe-button disabled danger>Danger disabled</pfe-button>
+  <pfe-button disabled warning>Warning disabled</pfe-button>
 
   <h3>Link variant</h3>
-  <pfe-button disabled variant="link" icon="plus-circle"><button>Link disabled</button></pfe-button>
-  <pfe-button disabled variant="link" icon="link" icon-position="right"><button>Link disabled</button></pfe-button>
-  <pfe-button disabled variant="link" inline><button>Inline link disabled</button></pfe-button>
-  <pfe-button disabled variant="link" danger><button>Danger link disabled</button></pfe-button>
+  <pfe-button disabled variant="link" icon="plus-circle">Link disabled</pfe-button>
+  <pfe-button disabled variant="link" icon="link" icon-position="right">Link disabled</pfe-button>
+  <pfe-button disabled variant="link" inline>Inline link disabled</pfe-button>
+  <pfe-button disabled variant="link" danger>Danger link disabled</pfe-button>
 
   <h3>Plain button</h3>
   <pfe-button disabled plain>
-    <button aria-label="Action">
-      <svg fill="currentColor" height="1em" width="1em" viewBox="0 0 352 512">
-        <path d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"></path>
-      </svg>
-    </button>
+    <svg aria-label="Action" fill="currentColor" height="1em" width="1em" viewBox="0 0 352 512">
+      <path d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"></path>
+    </svg>
   </pfe-button>
```

Comment:
Use offscreen text and aria-hidden on SVG. See https://www.sarasoueidan.com/blog/accessible-icon-buttons/